### ISSUE TITLE
Show fingerprint on server side window

### ISSUE THREAD

**Issue by Ayx03** [deskflow/deskflow#7924]
### Feature Request

It seems that fingerprint don't show anywhere in the server side window by default (I checked the logs too)
So I checked Edit -> Preferences and found the certificate location is `%localappdata%/Deskflow/tls/deskflow.pem`, so I opened `%localappdata%/Deskflow/tls/` and found the `local-fingerprint` file, but not every user is able to know where to find this and achieve this. I saw other issues saying 

So I think it's a great idea to add the fingerprint to the server side window like InputLeap which is the successor of barrier, and barrier is a deskflow fork.

I'm not saying that InputLeap is perfect, I've submitted two issues for them, their show fingerprint thing don't show the fingerprint until I go to the settings and click save, and their stop/quit don't even work, at least on Windows.

I also found a closed issue from 2019 (https://github.com/deskflow/deskflow/issues/5959) saying it's fixed in [v1.11.0-rc2](https://snapshots.symless.com/public/1.11.0-rc2?github=5959) but this issue just somehow still exists now. The link is invalid and I can't find the version in this GitHub repo either.

![image](https://github.com/user-attachments/assets/c4dc937c-7946-4b65-b6fe-0064810ffe7e)

**Comment by nbolton** [comment]
Yeah, I like that feature in Input Leap. It's a good idea. I think it started as a TLS vulnerability fix. Maybe this is related to #7806?

**Comment by sithlord48** [comment]
I have not added them to the view in that pr i only focused on the functionality but i agree that would be a good Pr to attach this to
